src/calendar/calendar-day.style.tsx

```diff
@@ -0,0 +1,180 @@
+import { Color } from "src/color";
+import { TextStyleHelper } from "src/text";
+import styled, { css } from "styled-components";
+import { VariantDay } from "./types";
+
+interface DayNumerProps {
+    variant: VariantDay;
```

Comment:
one question.

the classname also included?

Reply:
let me revisit the css styling. I missed out on this. I think it can be reworked so we don't have to use `className`, `data-cell`, `data-circle`. as Keith mentioned [here](https://github.com/LifeSG/react-design-system/pull/84#discussion_r1100966021), we should avoid passing down classnames and make use of props for styling.

Reply:
so the intention here is to style `Circle` based on whether this date is selected, ya?

instead of styling `Circle` indirectly from the parent which is `GrowDayCell`, you can pass an additional prop like `<Circle $selected={ /* logic to determine if date is selected */ } />`

now all the styles for `Circle` are in one place, which makes it easier to check and update in the future

Reply:
Yes and also not.

This calendar will be use in range selection later. The <LeftCell /> element handle Styling Left Side during responsive.

Left SideCell
Top, Bottom Dashed - start from LeftCell, so it wouldn't break in responsive. Because the LeftCell always will grow during responsive.

Reply:
removed data-attribute for selection from css.

replace with 
```
${props => {
  ${LeftCell} ${Circle} {}
}}
```